frontend/javascripts/messages.tsx

```diff
@@ -79,6 +80,8 @@ export const settingsTooltips: Partial<Record<keyof RecommendedConfiguration, st
     "Set the blend mode for the dataset. The additive mode (default) adds the data values of all color layers. In cover mode, color layers are rendered on top of each other so that the data values of lower color layers are hidden by values of higher layers. Cover mode enables reordering of color layers.",
   renderWatermark: "Show a WEBKNOSSOS logo in the lower-left corner of each screenshot.",
   antialiasRendering: "Antialias rendering (can impact performance)",
+  colorLayerOrder:
+    "Set the order in which color layers are rendered. This setting will only affect the cover blend mode.",
```

Comment:
```suggestion
    "Set the order in which color layers are rendered. This setting is only relevant if the cover blend mode is active.",
```

is a bit clearer in my opinion.